src/emcpy/__init__.py

```diff
@@ -5,8 +5,8 @@
 
 import os
 import emcpy.calcs
-import emcpy.utils
-import emcpy.stats
+import emcpy.utils.utils
```

Comment:
I don't particularly like this 'redundant' naming, can we add something to the `__init__` files so that all of the existing stuff stays `import emcpy.utils` but then there is `import emcpy.utils.dateutils` also?

Reply:
I think I figured out a fix.